src/redux/userpanels/userPanelsSlice.ts

```diff
@@ -40,10 +42,24 @@ export const userPanelsSlice = createSlice({
       // Add to front
       state.addedWebpanels.splice(0, 0, action.payload);
       return state;
+    },
+    addShowcomposerBookmark: (
+      state,
+      action: PayloadAction<{ address: string; port: number }>
+    ) => {
+      if (state.bookmarks.find((bookmark) => bookmark.title === 'Showcomposer')) {
+        return state; // Showcomposer bookmark already exists
+      } else {
+        const { address, port } = action.payload;
+        const src = `http://${address}:${port}/showcomposer`;
+        state.bookmarks.push({ title: 'Showcomposer', src });
+        return state;
+      }
```

Comment:
```suggestion
      }
      const { address, port } = action.payload;
      state.bookmarks.push({
        title: 'Showcomposer',
        src: `http://${address}:${port}/showcomposer`
      });
      return state;
    }
```

Reply:
@alexanderbock what is the official name?